### PR TITLE
Update MCR.cpp

### PR DESCRIPTION
**Pull request by pengyue-py**
check the code format

### CODE REVIEW

**Review by hahaBlizzard — COMMENTED** [review]
testing

**Review by hahaBlizzard — COMMENTED** [review]
看不懂

File: MCR.cpp [hunk 1]
```diff
@@ -56,3 +56,5 @@ cout << game[1][0] << " " << game[1][1] << " " << game[1][2] << endl;
 cout << game[2][0] << " " << game[2][1] << " " << game[2][2] << endl;
 return 0;
 }
+
+//需要修正代码格式
```
Comment (nicoleduuu) [inline MCR.cpp:60]: // Ln 38: cout << "Which cell to mark? i:[0..2], j:[0..2]: ";

Comment (nicoleduuu) [reply MCR.cpp:60]: 没搞懂

Comment (pengyue-py) [reply MCR.cpp:60]: 就是输出一行提示信息

Comment (nicoleduuu) [reply MCR.cpp:60]: 但我们只能在line56之后+comment，前面的55行都选不了

**Review by Zartist — COMMENTED** [review]
//Line 39: No scope check is done on the user's input

File: MCR.cpp [hunk 2]
```diff
@@ -35,7 +35,7 @@ false to true.
  cout << "Player 1: ";
  else
  cout << "Player 2: ";
- cout << "Which cell to mark? i:[1..3], j:[1..3]: ";
+ cout << "Which cell to mark? i:[1..3], j:[1..3]: "; // user alerts
  cin >> i >> j;
```
Comment (Zartist) [inline MCR.cpp:39]: if (i >= 0 && i <= 2 && j >= 0 && j <= 2) cin >> i >> j;

**Review by Zartist — COMMENTED** [review]
No scope check is done on the user's input

File: MCR.cpp (hunk 2 above)
Comment (Zartist) [inline MCR.cpp:39]: // Line 39: No scope check is done on the user's input

Comment (pengyue-py) [reply MCR.cpp:39]: > // Line 39: No scope check is done on the user's input

I've already merged this patch to the main menu,hahaha.So just let it go.

**Pull request merged**